src/utils/create-and-authenticate-user.ts

```diff
@@ -0,0 +1,35 @@
+import axios from 'axios'
+import { CreateUserService } from '../services/create-user-service'
+
+export async function createAndAuthenticateUser() {
```

Comment:
essa função é utilizada nos testes? se for, não é uma boa prática utilizar regras de negócio dentro de funções usadas em testes, seja chamar o service ou executar uma chamada pra própria api. Quando fazemos isso, o teste passa a depender de outras funcionalidades que podem quebrar e dar falsos negativos/positivos